launcher/spec/launch_spec.go

```diff
@@ -293,6 +303,30 @@ func (s *LaunchSpec) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+func (s *LaunchSpec) setAttestationServiceVars(unmarshaledMap map[string]string) error {
+	if attestationServiceAddr, ok := unmarshaledMap[attestationServiceAddrKey]; ok {
```

Comment:
Can we remove the attestationServiceAddrKey and the associated code? Our tests don't use them anymore and the MD key is blocked by GCE so customers shouldn't be using it.